spec/spec_helper.rb

```diff
@@ -1,5 +1,5 @@
-require 'simplecov'
-SimpleCov.start 
+# require 'simplecov'
```

Comment:
will have to reject the commented out simplecov when we merge

Reply:
Agree with Chee.